react/lib/src/components/rating/rating.tsx

```diff
@@ -1,49 +1,37 @@
-import type {RatingProps} from '@agnos-ui/react-headless/components/rating';
+import type {RatingProps, RatingDirectives, StarContext, RatingState} from '@agnos-ui/react-headless/components/rating';
 import {createRating} from '@agnos-ui/react-headless/components/rating';
 import {Slot} from '@agnos-ui/react-headless/slot';
 import {useWidgetWithConfig} from '@agnos-ui/react-headless/config';
+import {useDirective} from '@agnos-ui/react-headless/utils/directive';
 import React from 'react';
 
 export * from '@agnos-ui/react-headless/components/rating';
 
+function Star({star, state, directive}: {star: StarContext; state: RatingState; directive: RatingDirectives['starDirective']}) {
+	const starContainer = useDirective(directive, {index: star.index});
+	return (
+		<React.Fragment key={star.index}>
+			<span className="visually-hidden">({star.index < state.visibleRating ? '*' : ' '})</span>
+			<span ref={starContainer} className="au-rating-star">
```

Comment:
```suggestion
			<span ref={starContainer}>
```
`au-rating-star` is already added by the `starContainer` directive, right?